src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+$stopwatch-size: 80vmin;
+$stopwatch-center: $stopwatch-size / 2;
+$center-size: 5vmin;
+$center-center: $center-size / 2;
+$minutes-hand-width: 3vmin;
+$minutes-hand-heigh: 20vmin;
+$minutes-hand-x-center: $minutes-hand-width / 2;
+$seconds-hand-width: 1.5vmin;
+$seconds-hand-heigh: 38vmin;
```

Comment:
Similarly, there is a typo in the variable name `$seconds-hand-heigh`. It should be `$seconds-hand-height` to maintain consistency and clarity.